_posts/2025-03-25-code4lib-2025-reflections.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Code4Lib 2025"
+date: 2025-03-25T12:00:00
+author:
+    - Hillel Arnold
+    - Patrick Galligan
+    - Ima Oduok
+layout: post
+categories:
+    - Conferences/Education
+tags:
+    - code4lib
+    - technology
+    - conferences
+
+excerpt_separator: <!--more-->
+---
+
+Hillel, Ima, and I (Patrick) attended the annual Code4Lib conference in Princeton, NJ. As always, this year’s Code4Lib was well-programmed with many thoughtful presenters and presentations. We each saw lots of different themes in the conference, and came away with various takeaways about the importance of how we do our work, and ways to improve it. Our reflections below offer a brief insight into our experiences. 

```

Comment:
I didn't catch this before, but since we're all listed as authors is there a way to reframe this opening sentence so it's not first-person?

Reply:
How about third person for intro, and then first person for reflections. Something like:

`Hillel, Ima, and Patrick all attended this years annual Code4Lib conference in Princeton, NJ. As always, the conference was well-programmed with many thoughtful presenters and presentations. They each came away with different insights about their work, current trends in the field, and new and thoughtful ways to to engage with their community of practice. Their reflections below offer a brief insight into the experience.`